### PR TITLE
adjusting active session language

### PR DESCRIPTION
**Pull request by susanev**
### :nut_and_bolt: Description: What code changed, and why?

we often get questions about what it means to have an active session, this attempts to make it more clear

### :chains: Related Resources
none.

### :+1: Definition of Done
stuff is maybe more clear

### :athletic_shoe: How to Build and Test the Change
`make serve` from `components/automate-chef-io`

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [x] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### :camera: Screenshots, if applicable
session

### CODE REVIEW

File: components/automate-chef-io/content/docs/ldap.md [hunk 1]
```diff
@@ -36,8 +36,7 @@ require you to adjust the [IAM]({{< relref "iam-v2-overview.md" >}}) policy memb
 
 {{< info >}}
 Users who sign in via SAML will have a session time of 24 hours before needing to sign in again.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/ldap.md:38]: idk why we have this in the ldap docs, can we remove it?

File: components/automate-chef-io/content/docs/saml.md [hunk 2]
```diff
@@ -38,8 +38,7 @@ your existing configuration by following these steps:
 
 {{< info >}}
 Users who sign in via SAML will have a session time of 24 hours before needing to sign in again.
-Local, MSAD, and LDAP users will have their Chef Automate sessions refreshed while they maintain an active
-browsing session of the Chef Automate UI or until they sign out directly.
+Local, MSAD, and LDAP users will have their Chef Automate sessions refreshed while their Chef Automate UI browser window remains open or until they sign out directly.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/saml.md:41]: idk why we have this in the saml docs, can we remove it?

File: components/automate-chef-io/content/docs/ldap.md (hunk 1 above)
Comment (mjingle) [reply components/automate-chef-io/content/docs/ldap.md:38]: I'm pretty sure this warning is a hold-over from when SAML and LDAP were together in the same configuration document.

**Comment by susanev** [comment]
oh sorry one more thing @mjingle i think we should add the message to the local users page too https://automate.chef.io/docs/users/

and then remove the local users mention from the ldap page

could you do that? or do you want me to?

**Pull request merged**